docs/Base/1.2.ArtefatoGeneralista.md

```diff
@@ -9,3 +9,27 @@ Apresentação (em sala) explicando o artefato elaborado, com: (i) rastro claro
 A Wiki ou GitPages do Projeto deve conter um tópico dedicado ao Módulo Artefato Generalista, com 1 artefato que seja independente de metodologia adotada, histórico de versões, referências, e demais detalhamentos gerados pela equipe nesse escopo.
 
 Demais orientações disponíveis nas Diretrizes (vide Moodle).
+
+
+
+## 5W2H
+
+O 5W2H é um método para entender um problema e, portanto, compreender como ele pode ser solucionado. Basicamente, deve-se responder às seguintes perguntas:
+
+- Quem é afetado pelo problema?
+- O que é o problema?
+- Aonde o problema ocorre?
+- Quando o problema ocorre?
+- Por que ele ocorre?
+- Como solucioná-lo?
+- Quanto custa a solução?
+
+<img src="/imagens/5w2h.png" alt="Imagem do 5w2h" >
```

Comment:
Seria interessante também alterar a rota da imagem para rodar no GitPage

- alterar a parte
`<img src="/imagens/5w2h.png" alt="Imagem do 5w2h" >`
- pela versao web
 `<img src="https://raw.githubusercontent.com/UnBArqDsw2024-2/2024.2_G10_Recomendacao_Entrega_01/6d546a25e4d7862689f92638580a74532d6d00d5/docs/imagens/5w2h.png" alt="Imagem do 5w2h" >`

Reply:
claro! O único problema: no trecho "6d546a25e4d7862689f92638580a74532d6d00d5", isso indica o hash do commit [6d546a2](https://github.com/UnBArqDsw2024-2/2024.2_G10_Recomendacao_Entrega_01/pull/3/commits/6d546a25e4d7862689f92638580a74532d6d00d5). Portanto, vou colocar `/main`. Assim, quando o PR for aceito, o docsify vai olhar a versão web da main, não desse commit, caso a imagem do 5w2h mude por qualquer motivo. 

Devemos lembrar de fazer isso nas próximas versões também.